war/src/main/webapp/css/style.css

```diff
@@ -1891,3 +1891,28 @@ body.no-sticker #bottom-sticker {
+
+.remodal {
```

Comment:
All of this rules u should rewrite with bem. To use remodal inside of bem-block.